src/libs/Navigation/isReportOpenInRHP.ts

```diff
@@ -0,0 +1,17 @@
+import type {NavigationState} from '@react-navigation/native';
+import NAVIGATORS from '@src/NAVIGATORS';
+import SCREENS from '@src/SCREENS';
+
+const isReportOpenInRHP = (state: NavigationState | undefined): boolean => {
```

Comment:
Can we use the [useIsReportOpenInRHP](https://github.com/Expensify/App/blob/fe880cbac6388a74ea1aef9d48c60563448d2c14/src/hooks/useIsReportOpenInRHP.ts#L6) hook?

Reply:
@WojtekBoman could you check on this please? I'd rather not have two ways of checking if a report is open on RHP